save_command_strategies/cmdline.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+PANE_PID="$1"
+CPID=$(pgrep -P $PANE_PID)
```

Comment:
What does `CPID` stand for? Can we use expanded name?

Reply:
Child PID